cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -0,0 +1,5267 @@
+[
+  {
+    "abstract": {
+      "description": "Index"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
+      ""
+    ],
+    "date_published": "2023",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 1,
+      "size": 137529
+    },
+    "experiment": "JADE",
+    "files": [
+      {
+        "checksum": "adler32:46c1eb5f",
+        "size": 137529,
+        "uri": "root://eospublic.cern.ch///eos/opendata/jade/documentation/notes/0Index.pdf"
```

Comment:
Note the extra slash in the URI. The values should be "root://eospublic.cern.ch//eos/opendata/..." here and elsewhere, otherwise the file link would not work.